adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/ApacheExecParamParser.java

```diff
@@ -60,6 +60,8 @@ public ApacheExecParamParser(final CommandLineParser commandLineParser,
         availableOptions.addOption("h", "help", false, "Print this message");
         availableOptions.addOption("p", "proto", false, "Export validation results as" +
                 " proto");
+        availableOptions.addOption("x", "exclude", true, "Exclude files from " +
+                "semantic GTFS validation");
```

Comment:
Does this exclude files a) only from semantic validation, or b) also from data type validation?

The primary use case I know of for excluding files is having memory issues with huge shapes.txt files. In this case, you'd want b) if data type validation also costs memory. But, IIRC, the data type implementation is streaming, so you don't have to load the entire file into memory?

Thinking more, though, even with a streaming data type validation implementation - if you're not going to semantically validate the shapes.txt file, do you really want to pay the performance overhead of reading through an entire shapes.txt file for data type validation? I would think when most users say "ignore this file", they really mean ignore it entirely.

Reply:
Very good point, at present we are only handling case a). After discussion, the implementation will be carried out in 2 stages. The first one being "ignore this file for the whole process", the second one considers that a user could decide to ignore a file from a) but would still be able to execute a "csv validation" on the file. 

Keeping in mind that once a file has been excluded from "csv validation" it can not be semantically/type validated after.

Reply:
@lionel-nj make sure that we have open issues detailing our plans